docs/development/local-development.md

```diff
@@ -10,9 +10,9 @@ For example, if you think anything is unclear, or you think something needs to b
 
 You need the following dependencies for local development:
 
-- Git `>=2.33.0`
+- Git `>=2.45.1`
```

Comment:
Git recently released `2.45.1` to fix several CVEs, so might be good to bump our recommendation.

In any case, Renovate itself uses at least `2.45.1`.